crates/figma_import/src/transform_flexbox.rs

```diff
@@ -622,6 +622,7 @@ fn compute_background(
                 transform: Some(transform.to_2d()),
                 opacity: last_paint.opacity,
                 scale_mode: bg_scale_mode,
+                res_name: images.image_res(image_ref).map(|s| s.to_string()),
```

Comment:
not sure why you need a `to_string()` here. Maybe because `image_res()` returns `Option<&String>` instead of `Option<String>`? If so, you can probably change it to return `Option<String>`.

Reply:
Yes. That is right. HashMap get method returns Option<&String>. I moved this mapping to the image_res function.